x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleService.java

```diff
@@ -265,4 +270,8 @@ public void submitOperationModeUpdate(OperationMode mode) {
         clusterService.submitStateUpdateTask("ilm_operation_mode_update",
             new OperationModeUpdateTask(mode));
     }
+
+    private static boolean isStoppedOrClosed(State state) {
```

Comment:
this doesn't feel like it belongs in this class, but since it doesn't seem to be used elsewhere, I think 
it is reasonable to leave here for now. Might help to add javadocs explaining the nature of this check
and why using it in the assertion above is acceptable

Reply:
How about I make it a non static method with the name `isClusterServiceStoppedOrClosed` that takes no arguments? This would be in addition to javadocs

Reply:
sure!